benchmarks/hash_anchored_vs_plain.py

```
@@ -0,0 +1,428 @@
+
+    # Try to edit P6-P15 using OLD refs (from before edit)
+    for para_idx in target_paras:
+        old_ref = old_refs[para_idx]
+        marker = f"[P{para_idx:02d}]"
+        try:
+            doc2.replace("committee", "EDITED", paragraph=old_ref)
+            hash_correct += 1
+        except HashMismatchError:
+            hash_rejected += 1
+            print(f"    {old_ref}: targeted {marker} ← SAFELY CAUGHT (HashMismatchError)")
+        except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)